app/policies/comment_policy.rb

```diff
@@ -0,0 +1,19 @@
+class CommentPolicy < ApplicationPolicy
+  def create?
+    user.present?
+  end
+
+  def update?
+    record.user == user
+  end
+
+  def destroy?
+    record.user == user || user.admin?
+  end
+
+  class Scope < Scope
```

Comment:
I'm confused. Whaddya doing here? 🤔

Reply:
the same question, I'm trying to understand pundit, but it's a little confused to me too hehe

Reply:
I just understand the scope function with this post https://crypt.codemancers.com/posts/2018-07-29-leveraging-pundit/  so I think it's not necessary in this file, thanks :D